src/components/dialogs/limits/copy-limits.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { LIMITS, OPERATIONAL_LIMITS_GROUPS_1, OPERATIONAL_LIMITS_GROUPS_2 } from '../../utils/field-constants';
+import React, { useCallback } from 'react';
```

Comment:
remove the import React,
```suggestion
import { useCallback } from 'react';
```

Reply:
not resolved yet